base/inference.jl

```diff
@@ -3365,40 +3460,44 @@ function optimize(me::InferenceState)
     type_annotate!(me)
 
     # run optimization passes on fulltree
-    force_noinline = false
+    force_noinline = true
```

Comment:
We've been miscomputing this; but fortunately, since it only affects the cached=no case, they're haven't been any consumers of the data.